turbopack/crates/turbo-tasks-backend/src/backend/operation/aggregation_update.rs

```diff
@@ -50,7 +51,7 @@ pub fn is_root_node(aggregation_number: u32) -> bool {
 fn get_followers_with_aggregation_number(
     task: &impl TaskGuard,
     aggregation_number: u32,
-) -> Vec<TaskId> {
+) -> SmallVec<[TaskId; 4]> {
```

Comment:
Nit: maybe add a type alias for this so that you don't have to specify `4` everywhere?